_posts/2020-12-22-aws-antipatterns.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: AWS Antipatterns
+author: arto
+excerpt: >
+  Cloud platforms offer unprecedented agility, innovation, and speed. Yet I see the same mistakes being repeated over and over again. This blog post is a collection of some things people often get wrong - and some thoughts on how to dodge those mistakes.
```

Comment:
> get wrong

Comes across as negative :) how about: "This blog post is a collection of some of the pitfalls we unknowingly fall into."

Reply:
I like that idea! Fixing..